lib/Doctrine/DBAL/Schema/MySqlSchemaManager.php

```diff
@@ -176,25 +177,29 @@ protected function _getPortableTableColumnDefinition($tableColumn)
                 break;
         }
 
-        $length = ((int) $length == 0) ? null : (int) $length;
+        if ($this->_platform instanceof MariaDb102Platform) {
+            $columnDefault = $this->getMariaDb1027ColumnDefault($this->_platform, $tableColumn['default'] ?? null);
```

Comment:
According to `MySqlPlatform::getListTableColumnsSQL()`, the `default` column is always selected. What is `?? null` for?

Reply:
Things I do when I'm not 100% sure ;) New to the code base. Fixed !

Reply:
FYI, a test was made on the original code:

```
$columnDefault = (isset($tableColumn['default'])) ? $tableColumn['default'] : null;
```

Let me know if you want I revert to be sure.

Reply:
The original code is not always correct. Unless you see a reason why the check is needed, let's keep it simple.